src/test/java/com/libertymutualgroup/herman/aws/ecs/broker/s3/S3BucketEncryptionTest.java

```diff
@@ -1,99 +0,0 @@
-package com.libertymutualgroup.herman.aws.ecs.broker.s3;
```

Comment:
Adding a comment so this doesn't get forgotten if the S3BucketEncryption class gets brought back as per above.